### PR TITLE
Tweak: Added option to change the color of the navigation dots in carousel type widgets [ED-4970]

### PR DESCRIPTION
**Pull request by nisan250**
https://share.vidyard.com/watch/UmCja9w1Hhpn6EvzEqPAQN?

### CODE REVIEW

**Review by StyleShit — CHANGES_REQUESTED** [review]

**Review by StyleShit — APPROVED** [review]

**Review by matipojo — CHANGES_REQUESTED** [review]

File: includes/widgets/image-carousel.php [hunk 1]
```diff
@@ -497,10 +497,25 @@ protected function register_controls() {
 		);
 
 		$this->add_control(
-			'dots_color',
+			'dots_color_inactive',
```
Comment (matipojo) [inline includes/widgets/image-carousel.php:500]:
```suggestion
			'dots_inactive_color',
```

**Review by matipojo — APPROVED** [review]

File: includes/widgets/image-carousel.php (hunk 1 above)
Comment (nisan250) [reply includes/widgets/image-carousel.php:500]: Done

File: includes/widgets/image-carousel.php [hunk 2]
```diff
@@ -497,10 +497,25 @@ protected function register_controls() {
 		);
 
 		$this->add_control(
-			'dots_color',
+			'dots_inactive_color',
```
Comment (matipojo) [inline includes/widgets/image-carousel.php:500]: Please keep consistency
```suggestion
			'dots_color_inactive',
```

**Review by matipojo — CHANGES_REQUESTED** [review]

File: includes/widgets/image-carousel.php (hunk 2 above)
Comment (nisan250) [reply includes/widgets/image-carousel.php:500]: Done

**Pull request merged**